src/main/java/step2/domain/LottoWinningRules.java

```diff
@@ -0,0 +1,29 @@
+package step2.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class LottoWinningRules {
+    private static final int LOSE_PRIZE = 0;
+    private final Map<Integer, Integer> lottoPrize = new HashMap<>();
```

Comment:
여기도 `상수`와 `인스턴스 변수`의 구분을 위해 공백 라인을 추가해주시면 좋을 것 같습니다.

```suggestion
    private static final int LOSE_PRIZE = 0;
    
    private final Map<Integer, Integer> lottoPrize = new HashMap<>();
```